### ISSUE TITLE
Reimported 3D models not updating in Instanced Scene

### ISSUE THREAD

**Issue by IcyChills** [godotengine/godot#75970]
### Godot version

4.0.2.stable.mono.official [7a0977ce2]

### System information

Windows 10, Vulkan

### Issue description

Reimported an updated animated 3D model to Godot, old version of models remains in instanced scene. Saving and exiting, then restart Godot did not fix the issue. 

### Steps to reproduce

Reimported an updated animated 3D model to Godot which also belongs to an instanced scene.

### Minimal reproduction project

N/A

**Comment by AThousandShips** [comment]
Did you ever mark the models used as unique?

**Comment by AThousandShips** [comment]
Possibly duplicate of #68672 but hard to know with limited information on what is the exact issue and setup here

**Comment by IcyChills** [comment]
I refrained from making the instance locally available within the scene, as it was initially generated from the main scene. Consequently, I opted to discard the TPS controller I was constructing. Instead of generating an instance from the model in the main scene,

I created a fresh scene, imported the model into it, and then produced an instance of the model. As of now, the model is successfully updating on re-imports. However, I have yet to incorporate the scripts and IK setup into the controller. I will address this matter and provide further updates if necessary.

**Comment by Zireael07** [comment]
@IcyChills Link didn't post?

**Comment by IcyChills** [comment]
My apologies seems github didn't post it and I didn't notice. Power was off, so reply is late.

https://www.youtube.com/watch?v=VasHZZyPpYU

**Comment by IcyChills** [comment]
Here is a quick and rather sloppy made rigged 3D model for your convenience

https://filebin.net/ts0mvy27h59sgcw5

**Comment by and-rad** [comment]
Is this still a problem? The filebin is not accessible anymore and there is no MRP attached to this issue.

**Comment by IcyChills** [comment]
> Is this still a problem? The filebin is not accessible anymore and there is no MRP attached to this issue.

Sorry for the delayed response, didn't check github for a while. It is still a problem, changes made in Blender is not being updated when reimporting the model. Tested with blend and gltf formats.

Only workaround is to create the entire player scene from scratch with every update on the model. I'll have to create an MRP as the current project can't be shared anymore in its current state. I'll post a link once that is done.

**Comment by slumberface** [comment]
I am running into this issue on 4.1.1.

After iteratively updating a character's 3D model, eventually something happens where the model's inherited scene stops updating upon reimport.

Has anyone figured out a way to force the model in the inherited scene to start updating again? I've come across some older fixes for this sort've thing but they don't seem to work any longer.

**Comment by saminton** [comment]
I'm not sure if this is the same problem but this happens to me constantly when using the "editible children" feature. In this screenshot `house_1` and `house_2` are the same file :

![SCR-20240612-tjse](https://github.com/godotengine/godot/assets/15970757/c0898cc5-9472-46a9-8e00-bfb112ecd539)

I have noticed the scene stops updating the when the "inherited scene" icon is no longer there even though the children are still in yellow as if inherited from something 🤷.

When using glb files instead of blender files my child scene's children just disappear completely and I'm left with a empty Node3D instead.

This bug is my one major gripe with Godot so far. It's causing me to redo any modifications I might have done to the scenes children whenever it breaks, which is not all the time still but very often.

**Comment by SoulErasers** [comment]
Godot 4.3 I tried to recreate the Scene and copy all the nodes from previous scene, the problem still existed. It is updated in viewport but still playing the old animation in game. Still no solution till now?

**Comment by opponent019** [comment]
2025 I'm also finding this issue with Godot 4.5.dev4 and Blender 4.5 (_they're twinning!_) exporting in glb

**Comment by AThousandShips** [comment]
> 4.5.1.stable.mono this 6+ year old issue still exists and is completely unaddressed. Crazy.

How is 2023 6 years ago? If you have more details and can provide an MRP to help test this it would really help because the original MRP is lost

**Comment by MoogieOuttaMyDepth** [comment]
> > 4.5.1.stable.mono this 6+ year old issue still exists and is completely unaddressed. Crazy.
> 
> How is 2023 6 years ago? If you have more details and can provide an MRP to help test this it would really help because the original MRP is lost

I arrived here from having Googled the issue and originally finding it discussed on Reddit 6 years ago. Sorry for the confusion.

https://www.reddit.com/r/godot/comments/gv9k50/inherited_scene_doesnt_update_when_model/
